scripts/docs_version_pragma_check.sh

```diff
@@ -117,9 +117,10 @@ function findMinimalVersion()
         fi
     done
 
-    if [ -z version ]
+    if [ -z "$version" ]
     then
         printError "No release $sign$pragmaVersion was listed in available releases!"
+        exit 1
```

Comment:
And even if it got printed somehow, it would not interrupt the script so no one would notice the problem in CI.